### PR TITLE
Prune Tekton PipelineRun

### PR DESCRIPTION
**Pull request by mvmaestri**
#### What is this PR About?
This is a cronjob to prune tekton pipelineruns defining a number to keep

#### How do we test this?
helm template -f charts/cronjob-prune-tekton-pipelinerun/values.yaml charts/cronjob-prune-tekton-pipelinerun | oc apply -f-

cc: @redhat-cop/casl

### CODE REVIEW

File: charts/cronjob-prune-tekton-pipelinerun/templates/rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  labels:
+    template: cronjob-prune-tekton-pipelinerun
+  name: {{ .Values.namespace }}-tekton-pipelinerun-pruners
+roleRef:
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.job_service_account }}
+userNames:
+- system:serviceaccount:{{ .Values.namespace }}:{{ .Values.job_service_account }}
```
Comment (pabrahamsson) [inline charts/cronjob-prune-tekton-pipelinerun/templates/rolebinding.yaml]: Can you change the name of this file to `clusterrolebinding.yaml` to reflect that it is indeed a ClusterRoleBinding?

File: charts/cronjob-prune-tekton-pipelinerun/templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          serviceAccount: cleaner
+          containers:
+            - name: kubectl
+              image: ghcr.io/ctron/kubectl:latest
```
Comment (pabrahamsson) [inline charts/cronjob-prune-tekton-pipelinerun/templates/cronjob.yaml]: I would suggest using values for the image and tag similar to how other cronjobs in this repo is doing this.
```
image: {{ .Values.image }}:{{ .Values.image_tag }}
```
I would also suggest to use `quay.io/openshift/origin-cli:4.5` as the image/tag (it has both `oc` and `kubectl`).

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Thank you @mvmaestri, this looks good just two small comments inline.

File: charts/cronjob-prune-tekton-pipelinerun/templates/cronjob.yaml (hunk 2 above)
Comment (mvmaestri) [reply charts/cronjob-prune-tekton-pipelinerun/templates/cronjob.yaml]: ✅

File: charts/cronjob-prune-tekton-pipelinerun/templates/rolebinding.yaml (hunk 1 above)
Comment (mvmaestri) [reply charts/cronjob-prune-tekton-pipelinerun/templates/rolebinding.yaml]: ✅

**Comment by mvmaestri** [comment]
Done @pabrahamsson

**Review by pabrahamsson — APPROVED** [review]
Thank you @mvmaestri LGTM

**Pull request merged**